### PR TITLE
Guidance for formatting literals

### LINKED ISSUE #9070: Formatting guidelines: no rule for formatting literals (constants), nor any rule for attributes
This article describes using `camelCase` and `CamelCase` for must of the identifiers and/or classnames etc, but there is no mention of what the preferred style for literals is:

```f#
let [<Literal>] iceCube = "cold"   // camelCase
let [<Literal>] IceCube = "cold"  // CamelCase
let [<Literal>] ICECUBE = "cold"   // ALLCAPS
let [<Literal>] ICE_CUBE = "cold"   // ALL_CAPS
let [<Literal>] ice_cube = "cold"  // snake_case
```

Also, do we have any preference for placement of attributes? I think the rule ought to be that anything _but_ the `Literal` attribute should be placed on its own line and should not span multiple lines. Each attribute should be on its own line.

```f#
[<SomeAttribute>] 
class GetIce() ...

[<SomeAttribute>]
let Foo = "foo"

let [<Literal>] Foo = "foo"

// NOT OK (raises warning due to a compiler bug)
[<Literal>] let Foo = "foo"
[<Literal>] let Boo = "foo"
[<Literal>] let Hoo = "foo"
```

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 8c09996f-24ac-cf39-ef43-d7909cbc2b89
* Version Independent ID: 1efdd523-8552-6421-e149-74f76826ef8f
* Content: [F# code formatting guidelines](https://docs.microsoft.com/en-us/dotnet/fsharp/style-guide/formatting)
* Content Source: [docs/fsharp/style-guide/formatting.md](https://github.com/dotnet/docs/blob/master/docs/fsharp/style-guide/formatting.md)
* Product: **dotnet-fsharp**
* GitHub Login: @cartermp
* Microsoft Alias: **phcart**

### PR DESCRIPTION
**Pull request by cartermp**
Fixes #9070

### CODE REVIEW

File: docs/fsharp/style-guide/formatting.md [hunk 1]
```diff
@@ -646,3 +646,17 @@ type MyRecord =
 ```
 
 When applied to a parameter, they must be on the same line and separated by a `;` separator.
+
+## Formatting literals
+
+[F# literals](../language-reference/literals.md) using the `Literal` attribute should use place the attribute on its own line and camelCase naming:
```
Comment (rpetrusha) [inline docs/fsharp/style-guide/formatting.md]: The sentence doesn't make sense. I think it should be "...should place the attribute on its own line and use camelCase naming:"

**Review by rpetrusha — APPROVED** [review]
Once sentence needs to be revised, @cartermp. Once you do that, you can merge.

**Comment by abelbraaksma** [comment]
@cartermp, not sure whether it's best to raise a new issue on the typo, since this is already merged, but I'm happy to raise one if necessary.

**Comment by cartermp** [comment]
@abelbraaksma Please file an issue or feel free to submit a fix to the document directly. Both are fine.

**Pull request merged**